catalog/chatgpt-female-professor-bias.md

```diff
@@ -0,0 +1,39 @@
+---
+title: ChatGPT struggles to parse sentences where feminine pronouns refer to professors
```

Comment:
I think "struggles" is a bit weak here since it outright failed in this task.
```suggestion
title: ChatGPT was unable to correctly interpret sentences where the feminine pronoun referred to a professor
```